docs/using/installing/index.rst

```diff
@@ -340,13 +340,14 @@ The file ``cluster.key`` should be kept only by the cluster administrator; it do
 
 You are now able to generate authentication certificates for the control service and each of your nodes.
 To generate the control service certificate, run the following command from the same directory containing your authority certificate generated in the previous step.
-Replace ``example.org`` with the hostname of your control service node; this hostname should match the one you will give to REST API clients.
+Replace ``example.org`` with the hostname of your control service node; this hostname should match the hostname you will give to HTTP API clients.
+It should be a valid DNS name that HTTPS clients can resolve since they will use it as part of TLS validation.
+Using an IP address is not recommended as it may break some HTTPS clients.
```

Comment:
I'm concerned about this from a product perspective, because it makes owning a domain name and having administrative access to adding A records to it a prerequisite for trying out Flocker. Which clients have problems with this?

Also note that this is a merge conflict with https://github.com/ClusterHQ/flocker/pull/1477/files#diff-87e3ab44da633c377fd2a47b19dd220aR358